src/chess/uciloop.cc

```diff
@@ -122,13 +122,26 @@ bool ContainsKey(const std::unordered_map<std::string, std::string>& params,
   return params.find(key) != params.end();
 }
 
+std::string GetTimestamp() {
+  using namespace std::chrono;
+  auto time_now = system_clock::now();
+  auto ms =
+      duration_cast<milliseconds>(time_now.time_since_epoch()).count() % 1000;
+  auto timer = system_clock::to_time_t(time_now);
+  std::tm bt = *std::localtime(&timer);
```

Comment:
Maybe feed `std::localtime` directly to `put_time` instead of creating a temporary copy of the structure?